src/Mutator/Arithmetic/Assignment.php

```diff
@@ -54,7 +54,7 @@ public function mutate(Node $node)
         return new Node\Expr\Assign($node->var, $node->expr, $node->getAttributes());
     }
 
-    protected function mutatesNode(Node $node): bool
+    protected function mutatesNode(Node $node, array $mutatorSettings): bool
```

Comment:
What if we make the mutatorSettings par of the `MutatorConfig` class, and let it expose a `getSettings()` method? This means we don't have to pass mutatorSettings to every node that doesn't need it.

Reply:
done